projects/fsastorefrontlib/src/cms-components/checkout/components/personal-details-navigation/personal-details-navigation.component.ts

```diff
@@ -63,17 +59,13 @@ export class PersonalDetailsNavigationComponent implements OnInit, OnDestroy {
                 entry?.formData?.length > 0 ? entry?.formData[0].id : null;
               this.formService.submit(yFormData);
             }
-            const isProductConfigurable = (<FSProduct>cart?.entries[0]?.product)
-              ?.configurable;
             return this.formService.getSubmittedForm().pipe(
               map(formData => {
                 if (formData && formData.content) {
-                  if (!isProductConfigurable) {
-                    this.addressService.createAddressData(
-                      JSON.parse(formData.content),
-                      user
-                    );
-                  }
+                  this.addressService.createAddressData(
```

Comment:
Now, the address has to be saved even for banking products bcs the consignment process can't be finished without shipping address on order (it's a mandatory attribute from now on).
Address attributes will still be disabled for insurance and enabled for change in case of banking products but banking order will have delivery address :)